ui/src/components/Read/index.vue

```diff
@@ -347,27 +344,25 @@ function handleCodeData() {
     getTableData()
 }
 function autoRefreshTimeRange() {
-    if (isDatePicker.value) {
-        let json = yamlToJson(data.code)
-        const begin = new Date()
-        const end = new Date()
-        const oldBeginDate = new Date(json.data.timeRange.begin)
-        const oldEndDate = new Date(json.data.timeRange.end)
-        const interval = oldEndDate.getTime() - oldBeginDate.getTime()
+    let json = yamlToJson(data.code)
+    const begin = new Date()
+    const end = new Date()
+    const oldBeginDate = new Date(json.data.timeRange.begin)
+    const oldEndDate = new Date(json.data.timeRange.end)
+    const interval = oldEndDate.getTime() - oldBeginDate.getTime()
         
-        if (interval === last15Minutes.value) {
-            begin.setTime(begin.getTime() - last15Minutes.value)
-        } else if (interval === lastWeek.value) {
-            begin.setTime(begin.getTime() - lastWeek.value)
-        } else if (interval === lastMonth.value) {
-            begin.setTime(begin.getTime() - lastMonth.value)
-        } else if (interval === last3Months.value) {
-            begin.setTime(begin.getTime() - last3Months.value)
-        }
-        json.data.timeRange.begin = begin.toISOString()
-        json.data.timeRange.end = end.toISOString()
-        data.code = jsonToYaml(json.data).data
+    if (interval === last15Minutes) {
+        begin.setTime(begin.getTime() - last15Minutes)
+    } else if (interval === lastWeek) {
+        begin.setTime(begin.getTime() - lastWeek)
+    } else if (interval === lastMonth) {
+        begin.setTime(begin.getTime() - lastMonth)
+    } else if (interval === last3Months) {
+        begin.setTime(begin.getTime() - last3Months)
```

Comment:
There is a bug here

The "last xx" dynamic time range type cannot be determined by the interval. If a user selects a fixed time range such as "last 15 minutes", the component will not update the time range upon clicking the search button.

Reply:
Removed these code.